README.md

```diff
@@ -410,3 +410,6 @@ Table of Contents
 
 ## Data mining
   * http://www.monkeylearn.com/ - Text mining in the cloud, 1,000 queries for free per month.
+  * https://wit.ai/ - NLP for developers
+  * http://azure.microsoft.com/en-us/services/machine-learning/ - Predictive Analytics in the cloud
```

Comment:
nice, but I dont want to list every service that these huge clouds have - else amazon ML belongs here too etcetc, it never ends.

Am working on a web site for this data which might have tags till then I think it's best if we dont duplicate things in this format.

rest looks good, do you mind removing this one?